test/downstream/linearize.jl

```diff
@@ -19,6 +19,24 @@ lsys, ssys = linearize(sys, [r], [y])
 @test lsys.C[] == 1
 @test lsys.D[] == 0
 
+# With guesses
+@variables t x(t) [guess = 0] y(t) [guess = 0] u(t) [guess = 0] r(t)=0.0 [input = true]
```

Comment:
```suggestion
@variables t x(t) [guess = 0] y(t) [guess = 1] u(t) [guess = 0] r(t)=0.0 [input = true]
```

This is a much better test case which then highlights the issue with your proposed approach.

Reply:
Yes; will add non zero guesses.